### ISSUE TITLE
WPT tests of downloading from an iframe are timing out

### ISSUE THREAD

**Issue by pshaughn** [servo/servo#24973]
WPT html/semantics/embedded-content/the-iframe-element/iframe_sandbox_anchor_download_allow_downloads.sub.tentative.html and iframe_sandbox_navigation_download_allow_downloads.sub.tentative.html timeout.
In each case an anchor element inside an iframe is being .click()ed and then the test harness expects to be able to tell the download worked.
This could be a problem with clicking anchors in an iframe, or with download mechanisms, or with bad test harness expectations about what Servo does with downloads.

**Comment by pshaughn** [comment]
These tests are tentative, but the tentative aspect has to do with the sandboxing, not with the fact of downloading from an iframe in the first place. Since Servo isn't currently enforcing sandboxing in general, this timeout might signify something else. Looking at RUST_LOG=net output, the download_stash.py?token= hyperlink doesn't end up sending a request to the network; I'm going to try some more minimal cases to see if there's a real problem here.

**Comment by pshaughn** [comment]
Yes, there is a real problem! #25374 is the real not-related-to-tentative-behavior issue here.

**Comment by pshaughn** [comment]
If the click worked, we'd still probably end up failing this test for lack of any code that tests the "download" attribute.

**Comment by jdm** [comment]
These no longer timeout.
